.gitignore

```diff
@@ -29,6 +29,7 @@
 /core/sawtooth_core.egg-info/
 /families/settings/sawtooth_settings/protobuf/
 /families/supplychain/python/sawtooth_supplychain/protobuf
+/families/identity/python/sawtooth_identity/protobuf
```

Comment:
According to how the protogen script has been modified, it appears `/python` shouldn't be there.